spec/integ/megolm-integ.spec.ts

```diff
@@ -56,7 +71,14 @@ function createOlmSession(olmAccount, recipientTestClient) {
  *
  * @return {object} event
  */
-function encryptOlmEvent(opts) {
+function encryptOlmEvent(opts: {
+    sender?: string;
+    senderKey: string;
+    p2pSession: Olm.Session;
+    recipient: TestClient;
+    plaincontent?: object;
+    plaintype?: string;
+}): ToDeviceEvent {
```

Comment:
Same here for the doc-block. `object` vs `ToDeviceEvent`.

Can you also check the other places?